README.md

```diff
@@ -170,8 +218,9 @@ func main() {
 For Lambda support use version v1.0.0-rc.1 and higher
 ```
 
-Use AWS X-Ray SDK for Go to generate custom subsegments inside AWS Lambda handler function.
+Regarding Lambda integration, lambda will be responsible for generating segments for customers and send them. AWS X-Ray Go SDK will make sure there will be a FacadeSegment inside lambda context so that customers are able to instrument their application successfully for subsegments generation case which including `Capture`, `HTTP Client`, `AWS`, `SQL` and `Custom Subsegments` usage.
```

Comment:
1. "send them" -> "send them to X-Ray service" , probably this will be more clear.
2. nitpick : "including" should be changed to "includes"